plugin/loadbalance/README.md

```diff
@@ -31,3 +76,29 @@ Load balance replies coming back from Google Public DNS:
     forward . 8.8.8.8 8.8.4.4
 }
 ~~~
+
+Use weighted round robin strategy to load balance replies defined by the **file** plugin:
+
+~~~ corefile
+demo.plmt {
```

Comment:
Can we use `example.com` as the example zone?  I'm not sure what `plmt` and `plt` are referencing.

Reply:
Sure, it's done.

Reply:
Ack